### DISCUSSION TITLE
Animated.View usage with event ?

### DISCUSSION THREAD

**Discussion by fristyr** [software-mansion/react-native-reanimated#2874] [Q&A]
Let's say I have a little Animated.View with slide entering and exiting

```javascript
import Animated, {
  FadeOut,
  SlideInLeft,
  SlideOutRight
} from 'react-native-reanimated';

<Animated.View entering={SlideInLeft} exiting={SlideOutRight} >
   ...
</Animated.View>
```

And I want to Animate ( FadeOut ) this View again by pressing, let's say a button.
Without unmounting component to reach exiting animation, I need something like in the middle.
How can I achieve this?

**Comment by JasonCareter** [comment]
obviously, layout animation is triggered when layout changed (mounted 、unmounted、position、height、width )。 you can do this by create normal animation with useSharedValue、useAnimatedStyle and withTiming . Or you have to use  [Layout Transition](https://docs.swmansion.com/react-native-reanimated/docs/api/LayoutAnimations/layoutTransitions#example) and update the target view's layout on the specific event you defined .

**Reply by JasonCareter** [reply]
by the way , entering and exiting layout animation can only be triggered on mount and unmount stage.
